Polygon_mesh_processing/include/CGAL/Side_of_triangle_mesh.h

```diff
@@ -175,11 +177,12 @@ class Side_of_triangle_mesh
   , tree_ptr(&tree)
   , own_tree(false)
   {
+    box = tree.bbox();
   }
 
   ~Side_of_triangle_mesh()
   {
-    if (own_tree)
+    if (own_tree && tree_ptr!=nullptr)
```

Comment:
For the record, `delete nullptr;` is a no-op, and that is documented in the C++ norm.